pkg/compatibility/compatibility.go

```diff
@@ -201,14 +216,23 @@ func IsRHELCompatible(machineVersion, ocpVersion string) bool {
 	return false
 }
 
+func FindMajorMinor(version string) string {
+	splitVersion := strings.Split(version, ".")
+	return splitVersion[0] + "." + splitVersion[1]
+}
+
```

Comment:
```suggestion
func FindMajorMinor(version string) string {
	if splitVersion := strings.Split(version, "."); len(splitVersion) > 1 {
		return splitVersion[0] + "." + splitVersion[1]
	else {
		return splitVersion[0] + ".0"
	}
}
```

Reply:
I've been thinking about this.  Because the value ".0" would mean an actual dot-zero version exists.  I'm torn.  None of the values in the [rhcos_version_map](https://github.com/test-network-function/cnf-certification-test/blob/main/cnf-certification-test/platform/operatingsystem/files/rhcos_version_map) are missing their [1] positions right now so I guess we control the inputs in that way.

Reply:
If we encounter values in the rhcos_version_map in the future that are missing their values I'd say we can edit this at that time.  I'll keep it in mind.  👍

Reply:
I like proper error handling, `splitVersion[1]` could panic on certain inputs. Replace `.0` on `.x` or remove it. But the condition is right.